site_scons/state.py

```diff
@@ -74,6 +75,14 @@ def _getBinPath(binName, msg=None):
         log.debug("Found %s here %s" % (binName, binFullPath))
         return binFullPath
 
+def _getEupsVersion():
+    binName = 'pkgautoversion'
+    try:
+        version = subprocess.check_output(['pkgautoversion'])
```

Comment:
The output from the command may have some newlines and/or spaces, may be better to strip leading/trailing stuff.